systems/ukama-agent/profile/pb/profile.proto

```diff
@@ -0,0 +1,127 @@
+syntax = "proto3";
+
+option go_package = "pb/gen";
+
+package ukama.subscriber.profile.v1;
```

Comment:
if the agent is its own system, shouldn't the proto package namespace  be `package ukama.ukama_agent.profile.v1`;  instead of  `package ukama.subscriber.profile.v1;`

Reply:
fixed